content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -75,34 +77,34 @@ In addition to labels you [attach](#step-one-attach-label-to-the-node), nodes co
 with a standard set of labels. See [Well-Known Labels, Annotations and Taints](/docs/reference/labels-annotations-taints/) for a list of these.
 
 {{< note >}}
-The value of these labels is cloud provider specific and is not guaranteed to be reliable.
+The values of these labels are cloud provider specific and are not guaranteed to be reliable.
 For example, the value of `kubernetes.io/hostname` may be the same as the Node name in some environments
 and a different value in other environments.
 {{< /note >}}
 
 ## Node isolation/restriction
 
 Adding labels to Node objects allows targeting pods to specific nodes or groups of nodes.
-This can be used to ensure specific pods only run on nodes with certain isolation, security, or regulatory properties.
-When using labels for this purpose, choosing label keys that cannot be modified by the kubelet process on the node is strongly recommended.
+This can be used to ensure specific pods  that only run on nodes with certain isolation, security, or regulatory properties.
+When using labels for this purpose, choosing label keys which cannot be modified by the kubelet process on the node is strongly recommended.
 This prevents a compromised node from using its kubelet credential to set those labels on its own Node object,
 and influencing the scheduler to schedule workloads to the compromised node.
 
 The `NodeRestriction` admission plugin prevents kubelets from setting or modifying labels with a `node-restriction.kubernetes.io/` prefix.
 To make use of that label prefix for node isolation:
 
 1. Ensure you are using the [Node authorizer](/docs/reference/access-authn-authz/node/) and have _enabled_ the [NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction).
-2. Add labels under the `node-restriction.kubernetes.io/` prefix to your Node objects, and use those labels in your node selectors.
+2. Add labels under the `node-restriction.kubernetes.io/` prefix to your Node objects, and use those same labels in your node selectors.
 For example, `example.com.node-restriction.kubernetes.io/fips=true` or `example.com.node-restriction.kubernetes.io/pci-dss=true`.
 
 ## Affinity and anti-affinity
 
 `nodeSelector` provides a very simple way to constrain pods to nodes with particular labels. The affinity/anti-affinity
 feature, greatly expands the types of constraints you can express. The key enhancements are
 
-1. The affinity/anti-affinity language is more expressive. The language offers more matching rules
+1. The affinity/anti-affinity language is more expressive. This language offers more matching rules
    besides exact matches created with a logical AND operation;
-2. you can indicate that the rule is "soft"/"preference" rather than a hard requirement, so if the scheduler
+2. you can also indicate that the rule is "soft"/"preference" rather than a hard requirement, so if the scheduler
```

Comment:
Since there is no previous mention of an indication in this context, the sentence sounds right without the use of `also`